### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by ksafford** [neagle/gokibitz#198]
It would be nice to get a notification when an opponent posts a game record of a game that I played, but that I did not post. And to get notifications for comments on those games.

**Comment by Phelanpt** [comment]
I agree, was just about to post a feature request for this, or for a button that allows you to subscribe to a specific game.
